### PR TITLE
[T2A4][W11-A2] Zhi Jiang

### PR DESCRIPTION
**Pull request by zavfel**

### CODE REVIEW

**Comment by zavfel** [comment]
Ready for Review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -200,11 +200,29 @@
      */
     public static void main(String[] args) {
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:192]: Hope you ran tests after inlining functions.
Can you inline only methods to make code harder to read and navigate?

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -5,19 +5,10 @@
  * ====================================================================
  */
 
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.IOException;
+import java.io.*;
```
Comment (cmkumar87) [inline src/seedu/addressbook/AddressBook.java:8]: Let say you smelly code like this written y someone. Do you know how to import the right Classes in a jiffy using eclipse?

Comment (zavfel) [reply src/seedu/addressbook/AddressBook.java:8]: by pressing cmd+shift+o

**Pull request closed without merging**